SQLitePCLRaw.bundle_sqlite3mc/SQLitePCLRaw.bundle_sqlite3mc.csproj

```diff
@@ -5,7 +5,7 @@
     <AssemblyName>SQLitePCLRaw.batteries_v2</AssemblyName>
     <RootNamespace>SQLitePCL</RootNamespace>
     <PackageId>$(MSBuildProjectName)</PackageId>
-    <Description>A SQLitePCLRaw bundle for SQLite3 Multiple Ciphers--an encryption extension for SQLite with support for multiple ciphers.</Description>
+    <Description>A SQLitePCLRaw bundle for SQLite3 Multiple Ciphers--an encryption extension for SQLite with support for multiple ciphers. NOTICE! This package is maintained seperately from the SQLitePCLRaw project.</Description>
```

Comment:
There is a minor spelling mistake: _sep[**e**]()rately_  should be _sep[**a**]()rately_.

Reply:
Oh weird. Why isn't my spellcheck working on this file... Thanks for catching this!